packages/common/src/store/account/selectors.ts

```diff
@@ -28,6 +27,9 @@ export const getAccountToCache = (state: CommonState) => ({
   collections: state.account.collections
 })
 
+export const getWalletAddresses = (state: CommonState) =>
```

Comment:
We need the wallet addresses in various places and they change on successful sign-in. I opted to store them in redux for now because that allows audius-query-based hooks to subscribe to the value and update when it changes, while also making it available to sagas and (importantly) _settable_ from the sign-in sagas in a way that will actually trigger components to update.

The other option, and something I would like to do long term, is to but the current wallets in high-level app context so that all hooks that need to change when the current user changes can just consume a context value and update appropriately.

Reply:
I think this is fine - i think it will be a _long_ time before we totally eliminate redux (if we ever do), but the caching stuff is really the stuff that's got to go. for this other top level state stuff... probably fine!